### PR TITLE
HW_08

### PR DESCRIPTION
**Pull request by andreikasmykou**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -48,6 +48,7 @@
     "sequelize": "4.38.0",
     "sequelize-cli": "4.0.0",
     "split": "1.0.1",
+    "swagger-ui-express": "^3.0.10",
```
Comment (slowpokex) [inline package.json]: Remove this arrow please

File: .eslintrc.js [hunk 2]
```diff
@@ -6,7 +6,7 @@ module.exports = {
     },
     "extends": "eslint:recommended",
     "parserOptions": {
-        "sourceType": "module"
+        "sourceType": "myModule"
```
Comment (slowpokex) [inline .eslintrc.js]: Revert to module

**Review by slowpokex — APPROVED** [review]

File: package.json (hunk 1 above)
Comment (andreikasmykou) [reply package.json]: Done

File: .eslintrc.js (hunk 2 above)
Comment (andreikasmykou) [reply .eslintrc.js]: Done

**Pull request merged**